### PR TITLE
Playwright tests for Query

### PR DESCRIPTION
**Pull request by tumms2021389**
Added playwright tests for Query component as Table

### CODE REVIEW

File: tests/e2e/Digv2/Query.spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+const config = require('../../config');
+const common = require('../../common');
+
+test.beforeEach(async ({ page }) => {
+  await page.setViewportSize({ width: 1720, height: 1080 });
+  await page.goto('http://localhost:3502/portal');
+});
+
+test.describe('E2E test', () => {
+  test('should login, create case and run different test cases for User Reference', async ({
```
Comment (vishalshrm539) [inline tests/e2e/Digv2/Query.spec.js:12]: Shouldn't it be Query instead of User Reference ?

**Comment by JEAxero** [comment]
@tumms2021389 With the updates to the strings that Vishal mentioned, this looks good. I'll approve once those have been updated.

**Comment by tumms2021389** [comment]
> @tumms2021389 With the updates to the strings that Vishal mentioned, this looks good. I'll approve once those have been updated.

modified

**Review by JEAxero — APPROVED** [review]
Thanks. Looks good to me.

**Pull request merged**